scripts/oem/set-guarded-fru-leds.sh

```diff
@@ -37,9 +37,15 @@ do
         echo "$line2" | grep "dimm" >/dev/null
         rc=$?
         if [ $rc -eq 0 ]; then
-            busctl set-property xyz.openbmc_project.Inventory.Manager \
-                "$line2" xyz.openbmc_project.State.Decorator.OperationalStatus \
-                Functional b false;
+            echo "$line2" | grep "/xyz/openbmc_project/inventory" >/dev/null
```

Comment:
This will always pass as the prefix will always be "/xyz/openbmc_project/inventory" for a fru path.
Just a dead check but causes no harm.